roles/openshift_node_group/tasks/sync.yml

```diff
@@ -72,7 +72,7 @@
   retries: 60
   delay: 10
 
-- name: Wait for sync DS to set annotations on all nodes
+- name: Wait for sync DS to set annotations on master nodes
```

Comment:
At this point only masters have been bootstrapped, so this check won't resolve 'missing infra nodes' issue